.github/labeler.yml

```diff
@@ -1,106 +1,155 @@
 # list of labels and minimatch globs to match to apply the label.
 
 CI:
-- any: ['.cirrus.yml']
-- any: ['.github/dependabot.yml']
-- any: ['.github/labeler.yml']
-- any: ['.github/workflows/*']
-- any: ['.appveyor.yml']
-- any: ['.codecov.yml']
+  - all:
+      - changed-files:
+          - any-glob-to-all-files:
+              - '.cirrus.yml'
+              - '.github/dependabot.yml'
+              - '.github/labeler.yml'
+              - '.github/workflows/*'
+              - '.appveyor.yml'
+              - '.codecov.yml'
 
 documentation:
-- all: ['*/doc/*']
+  - all:
+      - changed-files:
+          - any-glob-to-all-files:
```

Comment:
For the `documentation` label, I think using `any-glob-to-all-files` is correct because `all:` was used before this MR.